src/components/table/partial-styles/tabulator-paginator.scss

```diff
@@ -49,6 +51,18 @@
     }
 }
 
+.has-no-pagination {
+    .tabulator-tableHolder {
+        height: 100% !important;
+        max-height: 100% !important;
```

Comment:
Even if it is not recommended to use `!important`. I can see that I don't see other option to do it in other way

Reply:
Yes. It's such an isolated case. So it won't affect anything else.

Reply:
I don't think 
```
height: 100% !important;
max-height: 100% !important;
```
This values are correct when we have `the calculation holder tab` at the bottom. See the attached picture with the calculation holder tab` in the other comment